src/games-spa/src/components/Word/styles.ts

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import Button from '@material-ui/core/Button';
+
+type Card = {
+  cardType: number;
+};
+
+export const GameCardContainer = styled.div`
+  display: block;
+  width: 100%;
+  height: 100%;
+`;
+
+export const GameCard = styled(Button)``;
+
+export const RevealedGameCard: any = styled.div<Card>`
```

Comment:
Instead of `any` is there an explicit type we can return from styled.div?

Reply:
I'm sure there is and I'll look into it more. I have never worked with Typescript before so definitely going to be a learning curve. Any help or direction is welcome! Otherwise I'll do some research and find the best approach.

Reply:
I removed the explicit `: any` and it looks like it implicitly returns the correct type, `StyledComponent<"div", any, Card, never>`. Just axe the `: any` and call it good.

![image](https://user-images.githubusercontent.com/5422827/106962636-50b78a00-6737-11eb-8467-adf4d8eeb684.png)

Reply:
Ah ok, so I was just making it harder than it needed to be haha. Thanks for looking into that!